src/DecoratorManifold.jl

```diff
@@ -69,7 +69,8 @@ end
 macro decorator_transparent_fallback(ex)
     return esc(quote @decorator_transparent_fallback :intransparent ($ex) end)
 end
-macro decorator_transparent_fallback(fallback_case, ex)
+macro decorator_transparent_fallback(fallback_case, input_ex)
+    ex = macroexpand(ManifoldsBase, input_ex)
     if ex.head == :function || ex.head == :(=) #complete or inline function
```

Comment:
With the expansion, the `||` case can be omitted.
```suggestion
    if ex.head == :function
```

Reply:
Oh and the following error message has to be adapted accordingly, too (same for the `_function` case.

Reply:
I think there is was a misunderstanding. `@inline` functions and functions defined using `=` instead of the `function` keyword are two different things. `macroexpand` doesn't touch `=`.

Reply:
Oh, then I had the wrong term in mind. Still, for functions defined with `=` `_fallback` does not work (neither does `_function`, but there I removed that and extended the error message).

Reply:
I think not supporting `=` functions is OK there.

Reply:
I think so, too. Just also adapt the error message :)